src/nwb/NWBFile.hpp

```diff
@@ -124,6 +127,9 @@ class NWBFile
   const std::string identifierText;
   std::shared_ptr<BaseIO> io;
   static std::vector<SizeType> emptyContainerIndexes;
+
+  std::string description;
+  std::string dataCollection;
```

Comment:
Since these are being written to the file, we should not save these on the NWBFile but instead make them accessible for read later.

Reply:
update to not save on the NWBFile